opnieuw/util.py

```diff
@@ -0,0 +1,17 @@
+from typing import Iterator, Optional, ContextManager
+
+from .retries import DoCall, RetryState, Action, replace_retry_state
+
+
+class NoRetryState(RetryState):
+    def __iter__(self) -> Iterator[Action]:
+        yield DoCall()
+
+
+def no_retries(namespace: Optional[str] = None) -> ContextManager[None]:
+    """
+    Contextmanager that disables retries for all `retry` and
```

Comment:
This function does not actually behave like a context manager itself.
```suggestion
    Returns a contextmanager that disables retries for all `retry` and
```